show/bgp_frr_v6.py

```diff
@@ -36,3 +36,34 @@ def neighbors(ipaddress, info_type):
     info_type = "" if info_type is None else info_type
     command = 'sudo vtysh -c "show bgp ipv6 neighbor {} {}"'.format(ipaddress, info_type)
     run_command(command)
+
+# 'network' subcommand ("show ipv6 bgp network")
+@bgp.command()
+@click.argument('ipaddress', metavar='[IPADDRESS|PREFIX]', required=False)
```

Comment:
Suggest to change metavar to match help text in Command Reference (I prefer the case/phrasing). I.e., `metavar='[<ipv6-address>|<ipv6-prefix>]`

Reply:
fixed.